### PR TITLE
chore: fixed remaining jsdoc/check-tag-names complaints

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
**Context**: Part of part of #648, porting old [TSLint](https://palantir.github.io/tslint) rules to [ESLint](https://eslint.org/) equivalents. This mostly finishes the work from #65080 and #65315 in fixing complaints from [`jsdoc/check-tag-names`](https://github.com/gajus/eslint-plugin-jsdoc/blob/main/docs/rules/check-tag-names.md#user-content-check-tag-names-options-typed).

Next up will be replacing `no-redundant-jsdoc2` in the ESLint configuration. ~Performance is much worse (as @jakebailey predicted in https://github.com/microsoft/DefinitelyTyped-tools/issues/648#issuecomment-1503797891) and will need to be improved first, I'm guessing.~ Per https://github.com/microsoft/DefinitelyTyped-tools/issues/648#issuecomment-1556240887, that rule's performance is no longer a blocker.

I think the build failures are unrelated:

```plaintext
Error in next-pwa
Error: Errors in typescript@4.5 for external dependencies:
node_modules/next/types/index.d.ts(3,23): error TS2688: Cannot find type definition file for 'react/experimental'.
node_modules/next/types/index.d.ts(5,23): error TS2688: Cannot find type definition file for 'react-dom/experimental'.

    at testTypesVersion (/home/vsts/work/1/s/node_modules/@definitelytyped/dtslint/src/index.ts:235:11)
    at runTests (/home/vsts/work/1/s/node_modules/@definitelytyped/dtslint/src/index.ts:197:7)


Error in yeoman-test
Error: 
A module look-up failed, this often occurs when you need to run `npm install` on a dependent module before you can lint.

Before you debug, first try running:

   npm install --prefix /home/vsts/work/1/s/types/inquirer

Then re-run. Full error logs are below.

Errors in typescript@5.1 for external dependencies:
../inquirer/index.d.ts(16,28): error TS2307: Cannot find module 'rxjs' or its corresponding type declarations.
../inquirer/lib/prompts/base.d.ts(2,28): error TS2307: Cannot find module 'rxjs' or its corresponding type declarations.
../inquirer/lib/prompts/editor.d.ts(2,39): error TS2307: Cannot find module 'rxjs' or its corresponding type declarations.
../inquirer/lib/utils/events.d.ts(2,28): error TS2307: Cannot find module 'rxjs' or its corresponding type declarations.

    at testTypesVersion (/home/vsts/work/1/s/node_modules/@definitelytyped/dtslint/src/index.ts:235:11)
    at runTests (/home/vsts/work/1/s/node_modules/@definitelytyped/dtslint/src/index.ts:197:7)
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
         "d3-time": "^3.0.0",
         "d3-time-format": "^4.0.0",
         "danger": "^11.2.3",
+        "eslint-plugin-jsdoc": "^44.0.1",
```
Comment (JoshuaKGoldberg) [inline package.json:42]: This doesn't need to be checked in. I found it useful during development to turn the rule on & off.

```suggestion
```

**Review by johnnyreilly — APPROVED** [review]
This looks good - thanks Josh!  Not sure why the build is failing though. Is your fork up to date?

**Comment by JoshuaKGoldberg** [comment]
Thanks! It is, yes. 😕

**Comment by JoshuaKGoldberg** [comment]
I believe @sandersn is working in parallel on some of the impacted packages, so I'll hold off for a bit.

**Comment by sandersn** [comment]
I fixed next-pwa and couldn't repro the yeoman failure locally. Maybe try again and see if the latter was an npm failure?

**Comment by DangerBotOSS** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  **types/vscode/index.d.ts#L19** - The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.
  </td>
    </tr>
  </tbody>
</table>



Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## angular-route ([<kbd>unpkg</kbd>](https://unpkg.com/browse/angular-route@latest/))
was missing the following properties:
1. padStart
1. padEnd
1. trimLeft
1. trimRight
1. trimStart

<details>
<summary>as well as these 1 other properties...</summary>
<p>trimEnd</p>
</details>

## angular-ui-bootstrap ([<kbd>unpkg</kbd>](https://unpkg.com/browse/angular-ui-bootstrap@latest/))
was missing the following properties:
1. padStart
1. padEnd
1. trimLeft
1. trimRight
1. trimStart

<details>
<summary>as well as these 1 other properties...</summary>
<p>trimEnd</p>
</details>

## babel-plugin-react-html-attrs ([<kbd>unpkg</kbd>](https://unpkg.com/browse/babel-plugin-react-html-attrs@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'babel-plugin-react-html-attrs'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.
## blessed ([<kbd>unpkg</kbd>](https://unpkg.com/browse/blessed@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'blessed'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. Program
1. tput
1. Tput
1. unicode

<details>
<summary>as well as these 1 other properties...</summary>
<p>helpers</p>
</details>

## cqrs-domain ([<kbd>unpkg</kbd>](https://unpkg.com/browse/cqrs-domain@latest/))
was missing the following properties:
1. errors
## d3-array ([<kbd>unpkg</kbd>](https://unpkg.com/browse/d3-array@latest/))
was missing the following properties:
1. scan
## dcp-client ([<kbd>unpkg</kbd>](https://unpkg.com/browse/dcp-client@latest/))
was missing the following properties:
1. initSync
1. createConfigFragments
1. initcb
1. fetchSync
## dottie ([<kbd>unpkg</kbd>](https://unpkg.com/browse/dottie@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'dottie'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.
1. find
## expired-storage ([<kbd>unpkg</kbd>](https://unpkg.com/browse/expired-storage@latest/))
was missing the following properties:
1. version
1. author
1. gitUrl
## gulp ([<kbd>unpkg</kbd>](https://unpkg.com/browse/gulp@latest/))
was missing the following properties:
1. Gulp
## highland ([<kbd>unpkg</kbd>](https://unpkg.com/browse/highland@latest/))
was missing the following properties:
1. isUndefined
1. isFunction
1. isObject
1. isString
1. isArray

<details>
<summary>as well as these 19 other properties...</summary>
<p>setImmediate, of, fromError, tap, pipeline, series, streamifyAll, isUndefined, isFunction, isObject, isString, isArray, setImmediate, of, fromError, tap, pipeline, series, streamifyAll</p>
</details>

## humblebee__styled-components-breakpoint ([<kbd>unpkg</kbd>](https://unpkg.com/browse/humblebee__styled-components-breakpoint@latest/))
was missing the following properties:
1. getMediaShorthands
1. getMixin
1. mediaRules
1. getNextMedia
1. getSmallestMedia

<details>
<summary>as well as these 3 other properties...</summary>
<p>mediaTemplate, ruleTemplate, mediaWidthRule</p>
</details>

## is ([<kbd>unpkg</kbd>](https://unpkg.com/browse/is@latest/))
was missing the following properties:
1. a
1. type
1. hosted
1. instance
1. instanceof

<details>
<summary>as well as these 22 other properties...</summary>
<p>nil, undef, args, arraylike, bool, false, true, element, divisibleBy, int, maximum, minimum, ge, gt, le, lt, primitive, hash, base64, hex, symbol, bigint</p>
</details>

## jsts ([<kbd>unpkg</kbd>](https://unpkg.com/browse/jsts@latest/))
was missing the following properties:
1. abstractFunc
1. error
1. index
1. geomgraph
1. noding

<details>
<summary>as well as these 1 other properties...</summary>
<p>triangulate</p>
</details>

## loopback ([<kbd>unpkg</kbd>](https://unpkg.com/browse/loopback@latest/))
was missing the following properties:
1. version
1. registry
1. errorHandler
1. faviconFile
1. remoteMethod

<details>
<summary>as well as these 15 other properties...</summary>
<p>template, createModel, configureModel, findModel, getModel, getModelByType, createDataSource, memory, DataSource, Connector, Memory, Mail, Remote, GeoPoint, ValidationError</p>
</details>

## maptalks ([<kbd>unpkg</kbd>](https://unpkg.com/browse/maptalks@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'maptalks'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. symbolizer
1. Browser
1. Canvas
1. Promise

<details>
<summary>as well as these 8 other properties...</summary>
<p>JSONAble, GEOMETRY_COLLECTION_TYPES, GEOJSON_TYPES, DEFAULT_TEXT_SIZE, OverlayLayer, TileConfig, Curve, TextMarker</p>
</details>

## matter-js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/matter-js@latest/))
was missing the following properties:
1. Matter
## node-red__registry ([<kbd>unpkg</kbd>](https://unpkg.com/browse/node-red__registry@latest/))
was missing the following properties:
1. init
1. load
1. clear
1. registerType
1. get

<details>
<summary>as well as these 61 other properties...</summary>
<p>registerSubflow, getNodeInfo, getNodeList, getModuleInfo, getModuleList, getNodeConfigs, getNodeConfig, getNodeIconPath, getNodeIcons, enableNode, disableNode, addModule, removeModule, installModule, uninstallModule, cleanModuleList, installerEnabled, getNodeExampleFlows, getNodeExampleFlowPath, getModuleResource, checkFlowDependencies, registerPlugin, getPlugin, getPluginsByType, getPluginList, getPluginConfigs, exportPluginSettings, deprecated, init, load, clear, registerType, get, registerSubflow, getNodeInfo, getNodeList, getModuleInfo, getModuleList, getNodeConfigs, getNodeConfig, getNodeIconPath, getNodeIcons, enableNode, disableNode, addModule, removeModule, installModule, uninstallModule, cleanModuleList, installerEnabled, getNodeExampleFlows, getNodeExampleFlowPath, getModuleResource, checkFlowDependencies, registerPlugin, getPlugin, getPluginsByType, getPluginList, getPluginConfigs, exportPluginSettings, deprecated</p>
</details>

## numjs ([<kbd>unpkg</kbd>](https://unpkg.com/browse/numjs@latest/))
was missing the following properties:
1. config
1. dtypes
1. NdArray
1. ndarray
1. flip

<details>
<summary>as well as these 4 other properties...</summary>
<p>leakyRelu, log, remainder, rot90</p>
</details>

## onfleet__node-onfleet ([<kbd>unpkg</kbd>](https://unpkg.com/browse/onfleet__node-onfleet@latest/))
was missing the following properties:
1. initBottleneckOptions
1. initBottleneckOptions
## orientjs ([<kbd>unpkg</kbd>](https://unpkg.com/browse/orientjs@latest/))
was missing the following properties:
1. RecordID
1. RecordId
1. RID
1. RIDBag
1. Bag

<details>
<summary>as well as these 27 other properties...</summary>
<p>Server, Db, Pool, Statement, Query, transport, errors, CLI, utils, jsonify, types, RecordID, RecordId, RID, RIDBag, Bag, Server, Db, Pool, Statement, Query, transport, errors, CLI, utils, jsonify, types</p>
</details>

## react-big-calendar ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-big-calendar@latest/))
was missing the following properties:
1. components
## react-bootstrap-table ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-bootstrap-table@latest/))
was missing the following properties:
1. ButtonGroup
1. InsertModalBody
## react-bootstrap-typeahead ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-bootstrap-typeahead@latest/))
was missing the following properties:
1. useHint
1. Input
1. asyncContainer
1. useAsync
1. withAsync

<details>
<summary>as well as these 5 other properties...</summary>
<p>menuItemContainer, useItem, withItem, tokenContainer, withToken</p>
</details>

## react-motion ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-motion@latest/))
was missing the following properties:
1. stripStyle
1. reorderKeys
## react-widgets ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-widgets@latest/))
was missing the following properties:
1. DatePicker
1. TimePicker
1. setNumberLocalizer
1. setDateLocalizer
1. setLocalizers

<details>
<summary>as well as these 1 other properties...</summary>
<p>utils</p>
</details>

## rpio ([<kbd>unpkg</kbd>](https://unpkg.com/browse/rpio@latest/))
was missing the following properties:
1. SOC_BCM2835
1. SOC_SUNXI
1. SOC_BCM2835
1. SOC_SUNXI
## selenium-webdriver ([<kbd>unpkg</kbd>](https://unpkg.com/browse/selenium-webdriver@latest/))
was missing the following properties:
1. ThenableWebDriver
## square-connect ([<kbd>unpkg</kbd>](https://unpkg.com/browse/square-connect@latest/))
was missing the following properties:
1. CatalogV1Id
1. Error
1. GetBankAccountByV1IdResponse
1. OrderFulfillmentPickupDetailsScheduleType
1. V1AdjustInventoryRequest

<details>
<summary>as well as these 163 other properties...</summary>
<p>V1AdjustInventoryRequestAdjustmentType, V1ApplyFeeRequest, V1ApplyModifierListRequest, V1BankAccount, V1BankAccountType, V1CashDrawerEvent, V1CashDrawerEventEventType, V1CashDrawerShift, V1CashDrawerShiftEventType, V1Category, V1CreateCategoryRequest, V1CreateDiscountRequest, V1CreateEmployeeRoleRequest, V1CreateFeeRequest, V1CreateItemRequest, V1CreateModifierListRequest, V1CreateModifierOptionRequest, V1CreatePageRequest, V1CreateRefundRequest, V1CreateRefundRequestType, V1CreateVariationRequest, V1DeleteCategoryRequest, V1DeleteDiscountRequest, V1DeleteFeeRequest, V1DeleteItemRequest, V1DeleteModifierListRequest, V1DeleteModifierOptionRequest, V1DeletePageCellRequest, V1DeletePageRequest, V1DeleteTimecardRequest, V1DeleteTimecardResponse, V1DeleteVariationRequest, V1Discount, V1DiscountColor, V1DiscountDiscountType, V1Employee, V1EmployeeRole, V1EmployeeRolePermissions, V1EmployeeStatus, V1Fee, V1FeeAdjustmentType, V1FeeCalculationPhase, V1FeeInclusionType, V1FeeType, V1InventoryEntry, V1Item, V1ItemColor, V1ItemImage, V1ItemType, V1ItemVisibility, V1ListBankAccountsRequest, V1ListBankAccountsResponse, V1ListCashDrawerShiftsRequest, V1ListCashDrawerShiftsResponse, V1ListCategoriesRequest, V1ListCategoriesResponse, V1ListDiscountsRequest, V1ListDiscountsResponse, V1ListEmployeeRolesRequest, V1ListEmployeeRolesResponse, V1ListEmployeesRequest, V1ListEmployeesRequestStatus, V1ListEmployeesResponse, V1ListFeesRequest, V1ListFeesResponse, V1ListInventoryRequest, V1ListInventoryResponse, V1ListItemsRequest, V1ListItemsResponse, V1ListLocationsRequest, V1ListLocationsResponse, V1ListModifierListsRequest, V1ListModifierListsResponse, V1ListOrdersRequest, V1ListOrdersResponse, V1ListPagesRequest, V1ListPagesResponse, V1ListPaymentsRequest, V1ListPaymentsResponse, V1ListRefundsRequest, V1ListRefundsResponse, V1ListSettlementsRequest, V1ListSettlementsRequestStatus, V1ListSettlementsResponse, V1ListTimecardEventsRequest, V1ListTimecardEventsResponse, V1ListTimecardsRequest, V1ListTimecardsResponse, V1Merchant, V1MerchantAccountType, V1MerchantBusinessType, V1MerchantLocationDetails, V1ModifierList, V1ModifierListSelectionType, V1ModifierOption, V1Money, V1Order, V1OrderHistoryEntry, V1OrderHistoryEntryAction, V1OrderState, V1Page, V1PageCell, V1PageCellObjectType, V1PageCellPlaceholderType, V1Payment, V1PaymentDiscount, V1PaymentItemDetail, V1PaymentItemization, V1PaymentItemizationItemizationType, V1PaymentModifier, V1PaymentSurcharge, V1PaymentSurchargeType, V1PaymentTax, V1PaymentTaxInclusionType, V1PhoneNumber, V1Refund, V1RefundType, V1RemoveFeeRequest, V1RemoveModifierListRequest, V1RetrieveBankAccountRequest, V1RetrieveBusinessRequest, V1RetrieveCashDrawerShiftRequest, V1RetrieveEmployeeRequest, V1RetrieveEmployeeRoleRequest, V1RetrieveItemRequest, V1RetrieveModifierListRequest, V1RetrieveOrderRequest, V1RetrievePaymentRequest, V1RetrieveSettlementRequest, V1RetrieveTimecardRequest, V1Settlement, V1SettlementEntry, V1SettlementEntryType, V1SettlementStatus, V1Tender, V1TenderCardBrand, V1TenderEntryMethod, V1TenderType, V1Timecard, V1TimecardEvent, V1TimecardEventEventType, V1UpdateCategoryRequest, V1UpdateDiscountRequest, V1UpdateEmployeeRequest, V1UpdateEmployeeRoleRequest, V1UpdateFeeRequest, V1UpdateItemRequest, V1UpdateModifierListRequest, V1UpdateModifierListRequestSelectionType, V1UpdateModifierOptionRequest, V1UpdateOrderRequest, V1UpdateOrderRequestAction, V1UpdatePageCellRequest, V1UpdatePageRequest, V1UpdateTimecardRequest, V1UpdateVariationRequest, V1Variation, V1VariationInventoryAlertType, V1VariationPricingType, V1EmployeesApi, V1ItemsApi, V1LocationsApi, V1TransactionsApi</p>
</details>

## three ([<kbd>unpkg</kbd>](https://unpkg.com/browse/three@latest/))
was missing the following properties:
1. BoxBufferGeometry
1. CapsuleBufferGeometry
1. CircleBufferGeometry
1. ConeBufferGeometry
1. CylinderBufferGeometry

<details>
<summary>as well as these 14 other properties...</summary>
<p>DodecahedronBufferGeometry, ExtrudeBufferGeometry, IcosahedronBufferGeometry, LatheBufferGeometry, OctahedronBufferGeometry, PlaneBufferGeometry, PolyhedronBufferGeometry, RingBufferGeometry, ShapeBufferGeometry, SphereBufferGeometry, TetrahedronBufferGeometry, TorusBufferGeometry, TorusKnotBufferGeometry, TubeBufferGeometry</p>
</details>

## tuya-panel-kit ([<kbd>unpkg</kbd>](https://unpkg.com/browse/tuya-panel-kit@latest/))
was missing the following properties:
1. Drawer
1. Wave
1. Diffusion
1. OfflineView
1. FullView

<details>
<summary>as well as these 5 other properties...</summary>
<p>Drawer, Wave, Diffusion, OfflineView, FullView</p>
</details>

## video.js ([<kbd>unpkg</kbd>](https://unpkg.com/browse/video.js@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'video.js'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
1. hooks_
1. hooks
1. hook
1. hookOnce

<details>
<summary>as well as these 52 other properties...</summary>
<p>removeHook, VERSION, options, getPlayers, getPlayer, getAllPlayers, players, getComponent, registerComponent, getTech, registerTech, use, middleware, browser, TOUCH_ENABLED, extend, mergeOptions, bind, registerPlugin, deregisterPlugin, plugin, getPlugins, getPlugin, getPluginVersion, addLanguage, log, createLogger, createTimeRange, createTimeRanges, formatTime, setFormatTime, resetFormatTime, parseUrl, isCrossOrigin, EventTarget, on, one, off, trigger, xhr, TextTrack, AudioTrack, VideoTrack, computedStyle, dom, url, mediaSources, MediaSource, URL, HlsHandler, HlsSourceHandler, Hls</p>
</details>

## wiredep ([<kbd>unpkg</kbd>](https://unpkg.com/browse/wiredep@latest/))
was missing the following properties:
1. config
## xstyled__styled-components ([<kbd>unpkg</kbd>](https://unpkg.com/browse/xstyled__styled-components@latest/))
was missing the following properties:
1. StyleSheetConsumer
1. StyleSheetContext
1. StyleSheetConsumer
1. StyleSheetContext
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 13f6c6eae10db6434262eac1574f6ba5c33c7cae
</p>

**Review by peterblazejewicz — APPROVED** [review]

**Comment by JoshuaKGoldberg** [comment]
Ready to merge

**Pull request merged**